effect/std/metadata.ts

```diff
@@ -7,7 +7,7 @@ import { rpcCall } from "./rpcCall.ts";
 
 export const metadata = effector(
   "metadata",
-  (rpc: EffectorItem<rpc.AnyClient>, blockHash?: EffectorItem<U.HashHexString>) => {
+  (rpc: EffectorItem<any>, blockHash?: EffectorItem<U.HashHexString>) => {
```

Comment:
Effects are in need of a refactor, given the way that RPC types flow through the beacon into any/all calls.